pavelib/pylint.py

```diff
@@ -0,0 +1,126 @@
+from __future__ import print_function
+import os
+import re
+
+from pavelib.utils.cmd import shell
+from pavelib.utils.envs import Env
+
+PACKAGE_DIRECTORIES = [
+    'djangoapps',
+    'lib',
+]
```

Comment:
`openedx/core` should be picked up as a submodule of `openedx`. As long as we tell pylint to check `openedx`, it should get everything in `core` as well.